orquesta/conducting.py

```diff
@@ -145,6 +145,9 @@ def remove_staged_task(self, task_id, route):
             if not any_items_running:
                 self.staged.remove(staged_task)
 
+    def get_sequence(self):
```

Comment:
Please change to `get_tasks`.  See other methods above on naming.  There are already methods that uses `get_task` and `get_tasks_*` for accessing self.sequence.  Also, please move this method to where the other related methods are located.